src/flexItem.test.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import TestUtils from 'react-addons-test-utils';
+import { IntlProvider } from 'react-intl';
+import FlexItem from './FlexItem';
+
+function intlRender(component) {
+	return TestUtils.renderIntoDocument(
+		<IntlProvider locale='en-US'>
+			{component}
+		</IntlProvider>
+	);
+}
```

Comment:
no need for `intlRender` here - there aren't any translated messages in the components being tested.